src/Microsoft.Health.SqlServer/Features/Client/SqlConnectionWrapperFactory.cs

```diff
@@ -29,10 +30,10 @@ public class SqlConnectionWrapperFactory
             _sqlConnectionFactory = sqlConnectionFactory;
         }
 
-        public async Task<SqlConnectionWrapper> ObtainSqlConnectionWrapperAsync(bool enlistInTransaction = false)
+        public async Task<SqlConnectionWrapper> ObtainSqlConnectionWrapperAsync(bool enlistInTransaction = false, CancellationToken cancellationToken = default)
```

Comment:
generally cancellationToken is not defaulted in the implementation, do you want to move the param has the first one?

Reply:
I will make it the first parameter and update references accordingly.

Reply:
I have made the fix and updated the PR.